csharp/README.md

```diff
@@ -22,6 +22,17 @@ Use `protoc` with the `--csharp_out` option to generate C# files in the specifie
 Include these in your C# project, and add a reference to the `Google.Protobuf` project. Currently
 there is no NuGet package for this, but we will be building one as soon as the API is stable.
 
+Supported platforms
```

Comment:
Yes, I suspected we don't have a supported platforms section, it's great you've added it. Thanks!